File: code/java/Branches.java
public class Branches {

	public static void main(String[] args) {
// BEGIN_CODE
		final int var = 3;

        if(var < 0) {
            System.out.println("Var is < 0");
        } else if(var == 0) {
            System.out.println("Var is 0");
        } else {
            System.out.println("Var is > 0");
        }

        switch(var) {
        case 1:
            System.out.println("Var is 1");
            break;
        case 2:
            System.out.println("Var is 2");
            break;
        case 3:
            System.out.println("Var is 3");
            break;
        default:
            System.out.println("Var is unknown");
        }

// END_CODE
	}
}

File: code/cpp/branches.cpp
#include <iostream>

#define unused __attribute ((unused))

using namespace std;

int main(unused int argc, unused char **argv) {
// BEGIN_CODE
	const int var = 3;

    if(var < 0) {
        cout << "Var is < 0" << endl;
    } else if(var == 0) {
        cout << "Var is 0" << endl;
    } else {
        cout << "Var is > 0" << endl;
    }

    switch(var) {
    case 1:
        cout << "Var is 1" << endl;
        break;
    case 2:
        cout << "Var is 2" << endl;
        break;
    default:
        cout << "Var is unknown" << endl;
    }

// END_CODE
}

